packages/rollup-full-node/test/app/web-rpc-handler.spec.ts

```diff
@@ -132,6 +132,42 @@ describe('Web3Handler', () => {
     })
   })
 
+  describe('the getBlockByHash endpoint', () => {
+    it('should return the same block that is returned by eth_getBlockByNumber', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const blockRetrievedByNumber = await httpProvider.getBlock('latest')
+      const blockRetrievedByHash = await httpProvider.getBlock(
+        blockRetrievedByNumber.hash
+      )
+
+      blockRetrievedByHash.should.deep.equal(blockRetrievedByNumber)
+    })
+
+    it('should return the same black as eth_getBlockByNumber even after another block is created', async () => {
+      const httpProvider = new ethers.providers.JsonRpcProvider(baseUrl)
+      const executionManagerAddress = await httpProvider.send(
+        'ovm_getExecutionManagerAddress',
+        []
+      )
+      const { timestamp } = await httpProvider.getBlock('latest')
```

Comment:
Add a check that this timestamp is the same as the timestamp from getBlockByNumber is the same as that of getBlockByHash